src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerMessage.cs

```diff
@@ -128,17 +128,17 @@ public static class LoggerMessage
         /// <param name="formatString">The named format string</param>
         /// <returns>A delegate which when invoked creates a log message.</returns>
         public static Action<ILogger, Exception?> Define(LogLevel logLevel, EventId eventId, string formatString)
-            => Define(logLevel, eventId, formatString, skipEnabledCheck: false);
+            => Define(logLevel, eventId, formatString, options: new LogDefineOptions() { SkipEnabledCheck = false});
```

Comment:
It would be nice if we didn’t have to allocate and object here just to throw it away. Maybe if we passed ‘null’ it should mean “use the default options”.